scout/adapter/mongo/query.py

```diff
@@ -715,3 +722,16 @@ def secondary_query(self, query, mongo_query, secondary_filter=None):
                 mongo_secondary_query.append({"cosmic_ids": {"$ne": None}})
 
         return mongo_secondary_query
+
+
+def _get_query_genotype(query):
+    """Query helper that returns the specific genotype selected by the user for a variantS query
+
+    Args:
+        query(dict): form dictionary with variantS query terms
+    """
+    q_value = query.get("genotypes")
+    if q_value == "other":
+        return {"$nin": ["0/1", "1/1", "0/0", "./."]}
```

Comment:
Most may at some point be interested also in phased GTs, but I guess we and the other centers are not quite there yet?

Reply:
No, let's keep it simpler!